crd/multitenancy/manifests/multitenancy.acn.azure.com_podnetworkinstances.yaml

```diff
@@ -80,6 +85,8 @@ spec:
               podnetwork:
                 description: Deprecated - use PodNetworks
                 type: string
+            required:
+            - defaultDenyACL
```

Comment:
also should disappear from here after making optional

Reply:
removed it